lua/rocks/config/internal.lua

```diff
@@ -36,6 +36,8 @@ local default_config = {
     lazy = false,
     ---@type boolean Whether to automatically add freshly installed plugins to the 'runtimepath'
     dynamic_rtp = true,
+    ---@type boolean Whether to re-generate plugins help pages after installation/upgrade
+    generate_help_pages = true,
```

Comment:
todo: we should add this to the user-facing `config` module (`config/init.lua`), too (CI will generate vimdoc from it).

Reply:
Woops, I don't know how I've missed this one, thank you!